lib/runners/cpp.js

```diff
@@ -20,6 +20,7 @@ module.exports.run = function run(opts, cb) {
   if (opts.setup) {
     var setupFile = util.codeWriteSync('cpp', opts.setup, dir, 'setup.h');
     args.push(setupFile);
+    if (opts.code) opts.solution = opts.code;
     opts.solution = '#include "setup.h"\n' + opts.solution;
```

Comment:
Fails because `opts.solution` is overwritten by `opts.code` in `shovel`. Fixing this now.